images/Dockerfile.kairos

```diff
@@ -10,6 +10,10 @@ RUN ORIG=/usr/bin/grub-editenv; DEST=/usr/sbin/grub2-editenv; [ -e $ORIG ] && [
 # here we add 100Mb aprox
 COPY --from=framework / /
 
+# TODO: temp change to test disabling selinux
+RUN sed -i 's/selinux=1/selinux=0/' /etc/cos/bootargs.cfg
+RUN echo "# edited by dimitris" >> /etc/cos/bootargs.cfg
```

Comment:
I guess this can be removed

Reply:
Sure, but the whole PR isn't going to be merged. It's just to test the effects of this one quickly: https://github.com/kairos-io/packages/pull/1022